### PR TITLE
TCC-15324 [ACO] Update documentation for Payments and Accounts

### PR DESCRIPTION
**Pull request by v-osadchuk**
## Brief description of what is being done:
Swagger documentation should be updated according to the changes that we are going to introduce into Payments and Accounts API. We should document new endpoints, document changes for existing endpoints (request parameters and responses) and document new validations or validations that were changed.

## Notes:
Should be updated with the precise information about end points changes and validations after new logic is implemented and fully tested.

Please also see related pull request [https://github.com/ccycloud/wordpress-website/pull/86](https://github.com/ccycloud/wordpress-website/pull/86).

### CODE REVIEW

File: src/reference.yaml [hunk 1]
```diff
@@ -7746,6 +8025,14 @@ paths:
               category: reference
               message: reference can not be longer than 255 character(s)
               params: ''
+            - code: charge_type_not_found
+              category: charge_type
+              message: Not available charge_type
+              params: ''
+            - code: invalid_charge_type
+              category: charge_type
+              message:  Invalid banking_charges supplied, must be in range [:ours, :shared]
```
Comment (Max43) [inline src/reference.yaml]: one thing here, please edit this message to be like that:
```Invalid banking_charges supplied, must be in range [ours, shared]```
just without dots
@v-osadchuk

Comment (v-osadchuk) [reply src/reference.yaml]: done, thanks @Max43

**Comment by rtriska** [comment]
@jonathancouchman if you are talking about Public API, then no. Public API changes are still in development stage. We will let you know when it is ready to be merged.
CC: @v-osadchuk

File: src/reference.yaml [hunk 2]
```diff
@@ -1080,6 +1080,274 @@ paths:
...
+            X-Request-Id:
+              type: string
+              description: A unique reference for the request.
+        default:
+          description: Unexpected error.
+          headers:
+            X-Request-Id:
+              type: string
+              description: A unique reference for the request.
+  '/accounts/{account_id}/payment_charges_settings/{id}':
```
[the remark this answers is not in the document]
Comment (rtriska) [reply src/reference.yaml]: @derikvercueil I will raise a topic on the Architecture guild to discuss conventions that we have to follow when designing new URL's.

**Comment by derikvercueil** [comment]
Closed. See #51

**Pull request closed without merging**